dana-bot/src/bot.js

```diff
@@ -983,6 +985,26 @@ function userRunEnd(msg) {
   var f = globalBot.buildPath + '/' + g.running.task + '/' + b.buildId + '.full.json';
   fs.writeFileSync(f, JSON.stringify(globalBot.currentRun));
 
+  var exportCmd = globalBot.config.configBot.runners[
+    Object.keys(globalBot.config.configBot.runners)[0]].env.exportCmd;
```

Comment:
Don't we know which runnner has executed the build? If we do, we could get the env of the actual runner instead of picking up the 1st one in the list.

Reply:
Name of runner can be retrieved from a serie (see serieDump()). But where this change is located (userRunEnd()) it seems to address all series, so maybe all runners (but I'm not yet familiar with the multi runners context). If so, there wouldn't be a current runner at this point.
So my guess is that, this exportCmd hook may not belong to any particular runner, but to the bot itself, so instead of putting it in a runner env, maybe I should move it to a higher level attribute of the config?
{ system: ...
  ip: ...
  port: ...
  runners: ...
  admin: ...
  exportCmd: ...
}
On the plus side, this would make it much simpler to reference.
var exportCmd = globalBot.config.configBot.exportCmd;

What do you think ?